testsupport/testng/src/main/java/com/adobe/testing/s3mock/testng/S3Mock.java

```diff
@@ -0,0 +1,72 @@
+/*
+ *  Copyright 2017-2018 Adobe.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.adobe.testing.s3mock.testng;
+
+import com.adobe.testing.s3mock.S3MockApplication;
+import com.adobe.testing.s3mock.testsupport.common.S3MockStarter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class S3Mock extends S3MockStarter {
+    private static S3Mock instance = null;
+
+    private S3Mock() {
+        super(null);
+        this.properties.putAll(defaultProps());
+        final Map<String, Object> properties = new HashMap<>();
+
+        String httpsPort =  System.getProperty(S3MockApplication.PROP_HTTPS_PORT);
```

Comment:
Isn't that working already? If not, we should fix that directly in `S3MockApplication`. After that, you wouldn't need to do anything else but call `super(null)` in this constructor.